lib/Trash/CollectivePageTrashItem.php

```diff
@@ -19,7 +20,13 @@ public function __construct(
 		IUser $user,
 		private string $mountPoint
 	) {
-		parent::__construct($backend, $originalLocation, $deletedTime, $trashPath, $fileInfo, $user);
+		[$major] = Util::getVersion();
+		if ($major < 30) {
+			parent::__construct($backend, $originalLocation, $deletedTime, $trashPath, $fileInfo, $user);
+		} else {
+			// *TODO* add $deletedby parameter to constructor
```

Comment:
```suggestion
			// *TOTO* Add support for deletedby to collectives trash backend table
```

But I'd suggest to also file an issue to not forget before the 30 final